forge-gui/res/cardsfolder/upcoming/dapper_shieldmate.txt

```diff
@@ -0,0 +1,7 @@
+Name:Dapper Shieldmate
+ManaCost:3 W
+Types:Creature Human Soldier
+PT:2/2
+K:etbCounter:Shield:1:no Condition:CARDNAME enters the battlefield with a shield counter on it.  (If it would be dealt damage or destroyed, remove a shield counter from it instead.)
```

Comment:
i wouldn't add the reminder text to the etbCounter effect

Reply:
I think I got them all